code/modules/antagonists/eldritch_cult/knowledge.dm/ash_lore.dm

```diff
@@ -0,0 +1,202 @@
+		C.IgniteMob()
+		C.adjust_fire_stacks(1)
+
+/datum/eldritch_knowledge/curse/corrosion
+	name = "Curse of Corrosion"
+	gain_text = "Cursed land, cursed man, cursed mind."
+	desc = "Curse someone for 2 minutes of vomiting and organ damage. Using a knife, a spill of blood, a heart, left arm and a right arm, and an item that the cursed person touched."
+	cost = 1
+	required_atoms = list(/obj/item/kitchen/knife,/obj/effect/decal/cleanable/blood,/obj/item/organ/heart,/obj/item/bodypart/l_arm,/obj/item/bodypart/r_arm)
+	next_knowledge = list(/datum/eldritch_knowledge/curse/blindness,/datum/eldritch_knowledge/spell/area_conversion)
+	timer = 2 MINUTES
+
+/datum/eldritch_knowledge/curse/corrosion/curse(mob/living/chosen_mob)
+	. = ..()
+	chosen_mob.apply_status_effect(/datum/status_effect/corrosion_curse)
+
+/datum/eldritch_knowledge/curse/corrosion/uncurse(mob/living/chosen_mob)
+	. = ..()
+	chosen_mob.remove_status_effect(/datum/status_effect/corrosion_curse)
+
+/datum/eldritch_knowledge/curse/paralysis
+	name = "Curse of Paralysis"
+	gain_text = "Corrupt their flesh, make them bleed."
+	desc = "Curse someone for 5 minutes of inability to walk. "
+	cost = 1
+	required_atoms = list(/obj/item/kitchen/knife,/obj/effect/decal/cleanable/blood,/obj/item/bodypart/l_leg,/obj/item/bodypart/r_leg,/obj/item/hatchet)
+	next_knowledge = list(/datum/eldritch_knowledge/curse/blindness)
+	timer = 5 MINUTES
+
+/datum/eldritch_knowledge/curse/paralysis/curse(mob/living/chosen_mob)
+	. = ..()
+	ADD_TRAIT(chosen_mob,TRAIT_PARALYSIS_L_LEG,MAGIC_TRAIT)
+	ADD_TRAIT(chosen_mob,TRAIT_PARALYSIS_R_LEG,MAGIC_TRAIT)
+
+/datum/eldritch_knowledge/curse/paralysis/uncurse(mob/living/chosen_mob)
+	. = ..()
+	REMOVE_TRAIT(chosen_mob,TRAIT_PARALYSIS_L_LEG,MAGIC_TRAIT)
+	REMOVE_TRAIT(chosen_mob,TRAIT_PARALYSIS_R_LEG,MAGIC_TRAIT)
+
+/datum/eldritch_knowledge/spell/cleave
+	name = "Blood Cleave"
+	gain_text = "At first i didn't know these instruments of war, but the priest told me to use them."
+	desc = "Gives AOE spell that causes massive bleeding. Be aware it can affect you aswell!"
+	cost = 1
+	spell_to_add = /obj/effect/proc_holder/spell/pointed/ash_cleave
+	next_knowledge = list(/datum/eldritch_knowledge/summon/raw_prophet,/datum/eldritch_knowledge/spell/area_conversion)
+
+/datum/eldritch_knowledge/ash_final
+	name = "Ashlord's rite"
+	gain_text = "The forgotten lords have spoken! The lord of ash have come! Fear the fire!"
+	desc = "Bring 3 corpses onto a transmutation rune, you will gain a powerful ability that is a high range spell that throws people away from you, sets them on fire and stuns them for 2 seconds. If anyone is already on fire it sleeps them for 10 seconds."
+	required_atoms = list(/mob/living/carbon/human)
+	cost = 3
+	route = "Ash"
+	var/finished = FALSE
+	var/list/trait_list = list(TRAIT_RESISTHEAT,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_NOFIRE,TRAIT_RADIMMUNE,TRAIT_PIERCEIMMUNE,TRAIT_NODISMEMBER,TRAIT_BOMBIMMUNE)
```

Comment:
ah, you DID give them a fire immunity passive... as the last tech in their tech tree

fair enough, I guess, but it'd still be nice if the fire immunity came earlier in a tech lower down on the tree, so that you can use your ashen blade to set people on fire without setting yourself on fire or going all the way to the end rite